server/service/src/invoice_line/stock_out_line/insert/generate.rs

```diff
@@ -110,8 +119,14 @@ fn generate_line(
         currency_rate,
         ..
     }: InvoiceRow,
+    default_pricing: ItemPrice,
 ) -> Result<InvoiceLineRow, RepositoryError> {
-    let total_before_tax = total_before_tax.unwrap_or(cost_price_per_pack * number_of_packs);
+    let cost_price_per_pack = line_cost_price_per_pack; // For now, we just get the cost price from the stock line
+
+    let sell_price_per_pack =
+        calculate_sell_price(line_sell_price_per_pack, pack_size, default_pricing);
+
+    let total_before_tax = total_before_tax.unwrap_or(sell_price_per_pack * number_of_packs); // CHECK ME, was cost_price_per_pack for some reason...
```

Comment:
thanks for noticing this !

Reply:
[4b7cdf9](https://github.com/msupply-foundation/open-msupply/pull/4756/commits/4b7cdf9042b08931d0a9f245037a6975c88e4826)